taxi/views.py

```diff
@@ -64,14 +67,14 @@ class CarDetailView(LoginRequiredMixin, generic.DetailView):
 
 class CarCreateView(LoginRequiredMixin, generic.CreateView):
     model = Car
-    fields = "__all__"
-    success_url = reverse_lazy("taxi:car-list")
+    form_class = CarForm
+    # success_url = reverse_lazy("taxi:car-list")
```

Comment:
Why you comment this?

Reply:
Because i want to navigate to detail page after creation